src/ckb/channel.rs

```diff
@@ -1590,19 +1677,41 @@ impl ChannelActorState {
             commitment_numbers: Default::default(),
             remote_commitment_points: vec![],
             local_shutdown_script: None,
-            local_shutdown_fee: None,
+            local_shutdown_fee_rate: None,
             remote_shutdown_script: None,
-            remote_shutdown_fee: None,
+            remote_shutdown_fee_rate: None,
             local_shutdown_signature: None,
             remote_shutdown_signature: None,
-            local_ckb_amount: DEFAULT_UDT_MINIMAL_CKB_AMOUNT,
-            remote_ckb_amount: DEFAULT_UDT_MINIMAL_CKB_AMOUNT,
+            local_reserve_ckb_amount,
+            remote_reserve_ckb_amount: 0,
 
             #[cfg(debug_assertions)]
             total_amount: value,
         }
     }
 
+    /// This function will be only called when the channel is accepted
+    fn check_reserve_amount(
+        &self,
+        local_reserved_amount: u64,
+        remote_reserved_amount: u64,
+    ) -> Result<(), ProcessingChannelError> {
+        let udt_type_script = &self.funding_udt_type_script;
+        let reserve_ckb_amount = if udt_type_script.is_some() {
+            DEFAULT_UDT_MINIMAL_CKB_AMOUNT
+        } else {
+            DEFAULT_CHANNEL_MINIMAL_CKB_AMOUNT
+        };
+        if local_reserved_amount < reserve_ckb_amount && remote_reserved_amount < reserve_ckb_amount
```

Comment:
Use `||` here?

Reply:
yes, thanks.
I'm trying to make all these kinds of check into a global utility function.